xcode/Subconscious/Shared/Services/DatabaseService.swift

```diff
@@ -320,12 +318,12 @@ final class DatabaseService {
         guard self.state == .ready else {
             throw DatabaseServiceError.notReady
         }
-        let suggestions = try database.execute(
+        let suggestions: [Suggestion] = try database.execute(
             sql: """
             SELECT id, title
             FROM memo
             ORDER BY modified DESC
-            LIMIT 25
+            LIMIT 5
```

Comment:
Intentional. It looked too visually busy with the new note suggestions to have so many recents.